boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts

```diff
@@ -309,12 +309,18 @@ i2s1: &flexcomm3 {
 	status = "okay";
 	mx25um51345g: mx25um51345g@2 {
 		compatible = "nxp,imx-flexspi-mx25um51345g";
-		size = <DT_SIZE_M(64)>;
+		size = <DT_SIZE_M(512)>;
```

Comment:
How about making `size` a const in the `nxp,imx-flexspi-mx25um51345g.yaml`.  Also, add a comment that this is in `bits`.  (personally I prefer to write this as `size = <DT_SIZE_M(64*8)>; /*size in bits */`